dev/build.image/profiles/webProfile11/features.xml

```diff
@@ -10,3 +10,29 @@
 <feature>sessionDatabase-1.0</feature>
 <feature>webCache-1.0</feature>
 <feature>requestTiming-1.0</feature>
+<feature>appAuthentication</feature>
+<feature>appSecurity</feature>
+<feature>beanValidation</feature>
+<feature>cdi</feature>
+<feature>concurrent</feature>
+<feature>enterpriseBeansLite</feature>
+<feature>expressionLanguage</feature>
+<feature>faces</feature>
+<feature>jdbc</feature>
+<feature>jsonb</feature>
+<feature>jsonp</feature>
+<feature>managedBeans</feature>
+<feature>pages</feature>
+<feature>persistence</feature>
+<feature>restfulWS</feature>
+<feature>restfulWSClient</feature>
+<feature>servlet</feature>
+<feature>websocket</feature>
+<feature>mpJwt</feature>
+<feature>mpTelemetry</feature>
+<feature>mpFaultTolerance</feature>
+<feature>mpOpenAPI</feature>
+<feature>mpRestClient</feature>
+<feature>mpHealth</feature>
+<feature>mpMetrics</feature>
+<feature>mpConfig</feature>
```

Comment:
```suggestion
```
You didn't have the MP ones in for jakarta ee 11 so they shouldn't be here either.